imports/client/ui/components/timeLine/TimeLine.jsx

```diff
@@ -48,59 +48,59 @@ export default class TimeLine extends React.Component {
   pauseOrClear = (event ) => {
 
 
-     window.clearInterval(this.timerForPlay);
-    if (this.playOrResumeTrig == 1){
+     clearInterval(this.timerForPlay);
+    if (this.state.playOrResumeTrig == true){
```

Comment:
you can remove the `== true`? It is implicit in JS. Also can you remove `Trig` from all the variable names ? Not sure it serves a purpose here.